delivery_deliverea/models/deliverea_request.py

```diff
@@ -73,28 +73,14 @@ def _check_error(self, res):
             if isinstance(error, list):
                 error = error[0]
             return_code = error.get("code")
-            message = error.get("message")
-            detail = error.get("detail", "")
             if return_code:
                 raise UserError(
-                    _("%(name)s: %(rcode)s %(message)s %(detail)s %(ccode)s")
-                    % {
-                        "name": _("Deliverea Error"),
-                        "rcode": return_code,
-                        "message": message,
-                        "detail": " ".join(
-                            ["%s: %s" % (key, value) for key, value in detail.items()]
-                        )
-                        if detail and not isinstance(detail, str)
-                        else detail or "",
-                        "ccode": "\n{}: {} {}".format(
-                            _("Carrier Error"),
-                            error.get("carrierCode"),
-                            error.get("carrierMessage"),
-                        )
-                        if error.get("carrierCode")
-                        else "",
-                    }
+                    _("%s")
```

Comment:
Do you plan to translate every response ?

Nevertheless, you should use translation function arguments instead.

Reply:
You are right, besides, the messages in response are already translated into Spanish (in my case), so it is not necessary to transalate one by one each message